### ISSUE TITLE
Add arguments to Bakery Setup and create-admin command

### ISSUE THREAD

**Issue by abdullahseba** [userfrosting/UserFrosting#808]
Could some sort of argument system be added to bakery?
Something that doesn't require user input like ` php bakery setup -args 0 localhost username` etc or ` php bakery setup -t 0 -h localhost -u -p`etc

**Comment by lcharette** [comment]
That's a good idea. Could be useful for automated tests too.

**Comment by lcharette** [comment]
Implemented in 32cf371. Just need to update the doc now.


Full example usage : 

```
php bakery setup -f --db_driver=mysql --db_name=UserFrosting --db_host=localhost --db_port=3306 --db_user=homestead --db_password=secret --smtp_host=foo --smtp_user=bar --smtp_password=secret
```

**Comment by lcharette** [comment]
Done !
